README.md

```diff
@@ -8,21 +8,32 @@ Please refer to [AppInsights-Home](https://github.com/Microsoft/ApplicationInsig
 
 Prerequisites
 -------------
-1. Java SDK 1.6 or higher
+1.  Java SDK 1.6 or higher
 
 Getting started
 ---------------
-1. Set JAVA_HOME environment variable to point to point to the [JDK installation directory]
-2. Recommended IDE is IntelliJ idea by using ‘Import Project’ rather than ‘Open Project’.
-3. You can use gradlew.bat on Windows to get gradle installed.
+1.  Set JAVA_HOME environment variable to point to point to the [JDK installation directory]
+2.  Recommended IDE is IntelliJ idea by using ‘Import Project’ rather than ‘Open Project’.
+3.  You can use gradlew.bat on Windows to get gradle installed.
 
 Using Eclipse IDE
 -----------------
-1. Install gradle from http://www.gradle.org/installation
-2. Add GRADLE_HOME/bin to your PATH environment variable
-3. In build.gradle add line [apply plugin: "eclipse"]
-4. In Eclipse used File->Import Existing Project in a workspace.
-5. Use [gradle build] to build the project from the command line.
+1.  Install gradle from http://www.gradle.org/installation
+2.  Add GRADLE_HOME/bin to your PATH environment variable
+3.  In build.gradle add line [apply plugin: "eclipse"]
```

Comment:
Is this the recommended way to work with Eclipse?
I'm a bit surprised that there's no simpler way, more IntelliJ like